### DISCUSSION TITLE
What of the follow three GGML types gives best preplexity. (Best Results) q5_1 or q5_K_M or q5_K_S

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#2831] [Q&A] [answered]
I have mostly been sticking to q5_1. I notice these other two use less ram. But are the slightly worse in perplexity/accuracy? 
Thanks. 
If possible link an actual benchmark if you can. 
Ooba now supports all of these three types right? I know the K variants were not supported at first. Or are they still not supported?

**Comment by berkut1** [comment]
I have tested.
In simple words - never use below Q_4, their are extremely perplexity. 
Use Q_6 if you can load at least 50% of GPU layer. Q_6 has low perplexity.

More information you can find here https://github.com/ggerganov/llama.cpp/pull/1684

**Comment by viperwasp** [comment]
That is super helpful! 
I liked this comment 
Quote 
"q5k_m which still is faster than old q5 but has nearly prepexity of fp16 or old q8."

If that is correct my go to Quantization should be Q5k_m for all the models I want. Do these work in Ooba yet? Or still not supported? 
Thanks a million berkut1 for the information.

**Reply by berkut1** [reply, marked as answer]
Ooba supports them, I've been using them for 2 weeks now.
